spring-integration-zip/src/test/java/org/springframework/integration/zip/splitter/UnZipResultSplitterTest-context.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:int="http://www.springframework.org/schema/integration"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+
+
+
+		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd">
+
+    <int:channel id="input"/>
+
+    <int:chain input-channel="input" output-channel="output">
+        <int:splitter>
+            <bean class="org.springframework.integration.zip.splitter.UnZipResultSplitter"/>
+        </int:splitter>
+    </int:chain>
+
+    <int:channel id="output">
+        <int:queue/>
+    </int:channel>
+
+</beans>
```

Comment:
Minor, of course, but annoying here on GH, and makes unnecessary changes during review locally, but any file must have a new line in the end.
That's also an IDE configuration option.